packages/xrpl/src/client/index.ts

```diff
@@ -564,6 +564,22 @@ class Client extends EventEmitter {
   /**
    * Tells the Client instance to connect to its rippled server.
    *
+   * @example
+   *
+   * Client.connect() establishes a connection between a Client object and the server.
+   *
+   * ```ts
+   * const { Client } = require('xrpl')
+   * const client = new Client('wss://s.altnet.rippletest.net:51233')
+   *   try {
+   *     await client.connect()
+   *     console.log('Connected to server')
+   *   } catch (error) {
+   *     console.log(`Failed to connect to server: ${error}`)
+   *   } 
```

Comment:
Recommend re-using the disconnect example because it highlights that you should have both with real code.
```suggestion
   * const { Client } = require('xrpl')
   * const client = new Client('wss://s.altnet.rippletest.net:51233')
   * await client.connect()
   * // do something with the client
   * await client.disconnect()
```